Userland/Applications/ThemeEditor/MainWidget.cpp

```diff
@@ -394,31 +403,31 @@ void MainWidget::build_override_controls()
     };
 }
 
-void MainWidget::add_property_tab(PropertyTab const& property_tab)
+ErrorOr<void> MainWidget::add_property_tab(PropertyTab const& property_tab)
 {
-    auto& scrollable_container = m_property_tabs->add_tab<GUI::ScrollableContainerWidget>(property_tab.title);
-    scrollable_container.set_should_hide_unnecessary_scrollbars(true);
+    auto scrollable_container = TRY(m_property_tabs->try_add_tab<GUI::ScrollableContainerWidget>(property_tab.title));
+    scrollable_container->set_should_hide_unnecessary_scrollbars(true);
 
-    auto properties_list = GUI::Widget::construct();
-    scrollable_container.set_widget(properties_list);
-    properties_list->set_layout<GUI::VerticalBoxLayout>();
+    auto properties_list = TRY(GUI::Widget::try_create());
+    scrollable_container->set_widget(properties_list);
+    TRY(properties_list->try_set_layout<GUI::VerticalBoxLayout>());
```

Comment:
This (and possibly other places) has to be `(void)`ed because the NNRP itself is `[[nodiscard]]` (this is what's causing the clang error).

Reply:
Thanks, I wish Clang and GCC would be more consistent on this!